### PR TITLE
Inheritance 2

### PR DESCRIPTION
**Pull request by xialiu1988**

### CODE REVIEW

File: inheritance/src/main/java/inheritance/Theater.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    public String toString(){
+        return String.format("The %s theatre is has %d good movies.",this.name,this.movies.size());
+    }
+
+    @Override
+    //get all the reviews about the theaters
+    public List<Review> getReviews() {
+        return this.theaterReviews;
+    }
+
```
Comment (evanslaton) [inline inheritance/src/main/java/inheritance/Theater.java:72]: extra line

File: inheritance/src/main/java/inheritance/Theater.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    //getters
+    public String getName() {
+        return this.name;
+    }
+
+    public List<String> getMovies() {
+        return this.movies;
+    }
+
+    public HashMap<String,List<Review>> getMovieReviews()
```
Comment (evanslaton) [inline inheritance/src/main/java/inheritance/Theater.java:24]: C# style

File: inheritance/src/main/java/inheritance/Theater.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+            this.movieReviews.put(movie,new ArrayList<>());
+        }
+    }
+
+    //add a movie
+    public void addMovie(String name){
+        movies.add(name);
+        movieReviews.put(name,new ArrayList<>());
+    }
+    //remove a movie
```
Comment (evanslaton) [inline inheritance/src/main/java/inheritance/Theater.java:54]: no empty line above this method

File: inheritance/src/test/java/inheritance/ShopTest.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class ShopTest {
+
+    @Test
+    public void CansetName() {
```
Comment (evanslaton) [inline inheritance/src/test/java/inheritance/ShopTest.java:12]: remember to use camelCase -> `canSetName()`

File: inheritance/src/main/java/inheritance/Theater.java [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+
+    public List<Review> getOneMovieReview(String movieName){
+         List<Review> rList = movieReviews.get(movieName);
+         return rList;
+    }
+
+    @Override
+    public void addReview(Review review) {
+            theaterReviews.add(review);
```
Comment (evanslaton) [inline inheritance/src/main/java/inheritance/Theater.java:81]: These lines look double-indented

File: inheritance/src/test/java/inheritance/ShopTest.java (hunk 4 above)
Comment (xialiu1988) [reply inheritance/src/test/java/inheritance/ShopTest.java:12]: ok. Will pay attention on that.

File: inheritance/src/main/java/inheritance/Theater.java (hunk 2 above)
Comment (xialiu1988) [reply inheritance/src/main/java/inheritance/Theater.java:24]: why? .. I think i am writing java...

**Pull request merged**